src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -0,0 +1,171 @@
+        30 => 'No date (format: YYYY-MM-DD)',
+        31 => 'Closing direction to the user',
+        32 => 'Not enough money',
+        33 => 'Missing phone number',
+        34 => 'Phone is in stop list',
+        35 => 'Not enough money',
+        36 => 'Can not obtain information about phone',
+        37 => 'Base Id is not set',
+        38 => 'Phone number already exists in this database',
+        39 => 'Phone number does not exist in this database',
+        999 => 'Unknown Error',
+    ];
+
+    public function __construct(string $login, string $password, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->login = $login;
+        $this->password = $password;
+        $this->from = $from;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('lightsms://%s/external/get/send.php?from=%s', $this->getEndpoint(), $this->from);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage;
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
+        $timestamp = time();
+        $data = [
+            'login' => $this->login,
+            'phone' => $this->escapePhoneNumber($message->getPhone()),
+            'text' => $message->getSubject(),
+            'sender' => $this->from,
+            'timestamp' => $timestamp,
+        ];
+        $data['signature'] = $this->generateSignature($data, $timestamp);
+        $endpoint = 'https://'.$this->getEndpoint().'/external/get/send.php?'.http_build_query($data);
+
+        $response = $this->client->request('GET', $endpoint);
+
+        if (Response::HTTP_OK !== $response->getStatusCode()) {
+            throw new TransportException('Unable to send the SMS: ', $response);
+        }
+
+        $content = $response->toArray(false);
+
+        // it happens if the host without www
+        if (isset($content['']['error'])) {
+            throw new TransportException('Unable to send the SMS: '.$this->getErrorMsg((int) $content['']['error']), $response);
+        }
+
+        if (isset($content['error'])) {
+            throw new TransportException('Unable to send the SMS: '.$this->getErrorMsg((int) $content['error']), $response);
+        }
+
+        $phone = $this->escapePhoneNumber($message->getPhone());
+        if (32 === (int) $content[$phone]['error']) {
+            throw new TransportException('Unable to send the SMS: '.$this->getErrorMsg((int) $content[$phone]['error']), $response);
+        }
+
+        if (0 === (int) $content[$phone]['error']) {
+            $sentMessage = new SentMessage($message, (string) $this);
+            $sentMessage->setMessageId($content[$phone]['id_sms']);
```

Comment:
```suggestion
            if (isset($content[$phone]['id_sms'])) {
                $sentMessage->setMessageId($content[$phone]['id_sms']);
            }
```

Because otherwise it can be undefined